pages/vi/vi-planet-installation-and-configuration.md

```diff
@@ -0,0 +1,163 @@
+  ```
+- Add Docker’s official GPG key: `curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -`
+- Use the following command to set up the stable repository
+  ```
+  sudo add-apt-repository \
+    "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+    $(lsb_release -cs) \
+    stable"
+  ```
+
+##### Install Docker CE
+
+- Update the apt package index: `sudo apt-get update`
+- Install the latest version of Docker CE: `sudo apt-get install docker-ce`
+- Verify that Docker CE is installed correctly by running the hello-world image: `sudo docker run hello-world`
+
+#### Install Docker Compose
+
+Follow the official [Install Docker Compose](https://docs.docker.com/compose/install/#install-compose) guide or the brief rundown below:
+
+- Note the latest release version number at https://github.com/docker/compose/releases/latest. e.g. `1.21.2`. You will need to replace `<latest-release-version-number>` with this in next command.
+- Run this command to download the latest version of Docker Compose:
+  ```
+  curl -L https://github.com/docker/compose/releases/download/<latest-release-version-number>/docker-compose-`uname -s`-`uname -m` -o /usr/local/bin/docker-compose
+  ```
+- Apply executable permissions to the binary: `sudo chmod +x /usr/local/bin/docker-compose`
+- Test the installation: `docker-compose --version`
+
+---
+
+## Planet Installation
+
+1. Launch Docker on your machine
+
+1. Go to your OLE project folder
+
+1. Clone the `planet` repository: `git clone https://github.com/open-learning-exchange/docker.git`
+
+1. Go to 'planet/docker' folder: `cd planet/docker`
+
+1. Pull the latest `planet` and its db-init Docker image
+
+  - `docker pull treehouses/planet:latest`
+  - `docker pull treehouses/planet:db-init`
+
+1. Run the following command to spawn your environment for the **first time**: `docker-compose -f planet.yml -p planet up -d --build`
+
+1. See if the docker containers are running: `docker ps`. You'll see your running container similar to this
+
+  ```
+  CONTAINER ID        IMAGE                       COMMAND                  CREATED             STATUS                      PORTS                                        NAMES
+  6ad5d3f2ba2b        treehouses/planet:latest    "/bin/sh -c 'sh ./do…"   38 seconds ago      Up 46 seconds               0.0.0.0:80->80/tcp                           planet_planet_1
+  e78eb9287454        treehouses/planet:db-init   "/bin/sh -c 'bash ./…"   38 seconds ago      Exited (0) 34 seconds ago                                                planet_db-init_1
+  3c2309e92dc6        treehouses/couchdb:2.1.1    "tini -- /docker-ent…"   39 seconds ago      Up 48 seconds               4369/tcp, 9100/tcp, 0.0.0.0:2200->5984/tcp   planet_couchdb_1
+  ```
+
+1. See log in action with `docker-compose -f planet.yml -p planet logs -f`, press 'CTRL+C' to exit logs view
+
+---
+
+## Planet Configuration
+
+Go to your community `planet` at <http://127.0.0.1>. You will be asked to:
```

Comment:
Hmmm...I see for this section using capital P might be confusing since the repo, directory, and docker are created with lowercase p.  I think we can leave this section lowercase because it's highlighted in a code block.